mmv1/templates/terraform/examples/cloud_run_service_interservice.tf.erb

```diff
@@ -57,7 +57,7 @@ resource "google_service_account" "default" {
 
 # [START cloudrun_service_interservice_private_service]
 resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>_private" {
-  name     = "<%= ctx[:vars]['private_service_name'] %>-private"
+  name     = "<%= ctx[:vars]['public_service_name'] %>-private"
```

Comment:
```suggestion
  name     = "<%= ctx[:vars]['public_service_name'] %>-private"
```

This seems off- the private service is using a variable called `public_service_name`?

Reply:
We could create a second variable here.

(I previously chose to use  a`-private` suffix instead.)

Reply:
For sure! Also, you'll want to create the variable in the `terraform.yaml` file as well. Right now `public_service_name` is an empty string.